consensus/pbft/batch.go

```diff
@@ -449,23 +401,17 @@ func (op *obcBatch) ProcessEvent(event events.Event) events.Event {
 				continue
 			}
 
-			if cert.prePrepare.RequestDigest == "" {
+			if cert.prePrepare.BatchDigest == "" {
 				// a null request
 				continue
 			}
 
-			if cert.prePrepare.Request == nil {
-				logger.Warningf("Batch replica %d found a non-null prePrepare with no request, ignoring")
-				continue
-			}
-
-			reqs := &RequestBlock{}
-			if err := proto.Unmarshal(cert.prePrepare.Request.Payload, reqs); err != nil {
-				logger.Warningf("Batch replica %d could not unmarshal request block: %s", op.pbft.id, err)
+			if cert.prePrepare.RequestBatch == nil {
+				logger.Warningf("Replica %d found a non-null prePrepare with no request batch, ignoring")
```

Comment:
Does this change modify the existing behavior? If it doesn't, do you agree that this issue is better off being addressed in a separate PR? If it does, I'll be glad to tackle it here.